bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -13,6 +16,9 @@ spec:
     metadata:
       labels: {{ include "postgresql-ha.labels" . | nindent 8 }}
         app.kubernetes.io/component: pgpool
+        {{- with .Values.pgpool.podLabels }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
Please, could you use `"postgresql-ha.tplValue"`? and have it as a template value too.